optimum/neuron/modeling_diffusion.py

```diff
@@ -842,9 +850,20 @@ def __init__(
     ):
         super().__init__(model, parent_model, config, neuron_config, DIFFUSION_MODEL_TEXT_ENCODER_NAME)
 
-    def forward(self, input_ids: torch.Tensor):
+    def forward(
+        self,
+        input_ids: torch.Tensor,
+        *args,
+        **kwargs,
+    ):
```

Comment:
I personally wouldn't just open up the door for any kwargs because there are actually other additional kwargs that are not being handled, and also I think we should assert the new args to make sure they are consistent with how it was compiled during tracing.
```suggestion
        output_attentions: Optional[bool] = None,
        output_hidden_states: Optional[bool] = None,
        return_dict: Optional[bool] = None,
    ):
        if attention_mask is not None:
            assert torch.equal(torch.ones_like(attention_mask), attention_mask), "attention_mask is expected to be only all ones"
        if output_hidden_states is not None:
            assert bool(self.text_encoder.config.output_hidden_states) == bool(output_hidden_states), "output_hidden_states is expected to be consistent with how it was compiled"

```

and this is like how I was going to PR it

Reply:
sorry for the wrong arg name 😅